src/main/java/movie/storage/dao/impl/CinemaHallDaoImpl.java

```diff
@@ -6,21 +6,26 @@
 import movie.storage.lib.Dao;
 import movie.storage.model.CinemaHall;
 import movie.storage.util.HibernateUtil;
+import org.apache.log4j.Logger;
 import org.hibernate.Session;
 import org.hibernate.Transaction;
 import org.hibernate.query.Query;
 
 @Dao
 public class CinemaHallDaoImpl implements CinemaHallDao {
+    private static final Logger logger = Logger.getLogger(CinemaHallDaoImpl.class);
+
     @Override
     public CinemaHall add(CinemaHall cinemaHall) {
+        logger.info("Add cinema hall");
```

Comment:
I would leave only logger "added" in other case we will have exception which will be logged anyway

Reply:
Agree